### ISSUE TITLE
switch state does not reflect valve state

### ISSUE THREAD

**Issue by steerage250** [sebr/bhyve-home-assistant#267]
I have a recently installed 6 station BHyve controller.  I have started trying to use HA to monitor (not control at present) its performance.  One program has 2 zones that water for 20 minutes.  Each minute HA checks the state of the switch, and if it is "on", it increments the zone runtime counter.

On the 2 most recent occasions:
1. The 1st zone status only stayed "on" for 1 minute (even though it watered for 20), the 2nd zone status behaved correctly.
2. The 1st zone status stayed "on" for 40 minutes (even though it watered for 20), the 2nd zone status behaved correctly

I would expect the zone status to be "on" for all the time the zone is watering - regardless of the method used to switch the zone on.

**Comment by bbobak86** [comment]
Also having this same issue or similar, any feed back on the cause or solution? Running a program through Orbit. Zone switch turns on and then turns off immediately. But the programs runs as it should.

**Comment by steerage250** [comment]
No, I have got no-where on getting the problem fixed.  What I have done is create some internal logic that creates a new, internal "Station 1 state" variable that goes to the Off state when Station 2 comes On.  That is - it hides the problem!

**Comment by sebr** [comment]
Unfortunately I only have the single zone BHyve devices, so I can't test or debug multi-station devices.
